sections/expansion_content.tex

```diff
@@ -74,122 +99,472 @@ \subsection*{Empowered Statistic Cards}
+                \draw (-1.5, -0.5) node {\includegraphics[width=0.15\linewidth]{\images/P.png}};
+                \draw (0, 0.6) node {\includegraphics[width=0.15\linewidth]{\images/P.png}};
+                \draw (0, -0.5) node {\includegraphics[width=0.15\linewidth]{\images/P.png}};
+                \draw (1.5, 0.6) node {\includegraphics[width=0.15\linewidth]{\images/P.png}};
+                \draw (1.5, -0.5) node {\includegraphics[width=0.15\linewidth]{\images/P.png}};
+                \draw (2.9, 1.7) node {\includegraphics[width=0.15\linewidth]{\images/N.png}};
+                \draw (2.9, -1.6) node {\includegraphics[width=0.15\linewidth]{\images/N.png}};
+                \draw (-3.1, 1.7) node {\includegraphics[width=0.15\linewidth]{\images/N.png}};
+                \draw (-3.1, -1.6) node {\includegraphics[width=0.15\linewidth]{\images/N.png}};
+            \end{tikzpicture}
+        \end{center}
+        \textbf{\footnotesize\textit{\textcolor{darkcandyapplered}{P -- player deployment zone}}}\\
+        \textbf{\footnotesize\textit{\textcolor{darkcandyapplered}{N -- Neutral Units deployment zone}}}\\
+        \bigskip
+
+    Based on the Scenario's Difficulty Level, take a number of \pagelink{Stack Tokens}{Stack Tokens} and place them randomly on up to four different Creature Bank Unit Cards:
+    \begin{itemize}
+        \item \textbf{Easy}: 1 token
+        \item \textbf{Normal}: 2 tokens
+        \item \textbf{Hard}: 3 tokens
+        \item \textbf{Impossible}: 4 tokens
+    \end{itemize}
+
+\end{multicols*}
 \end{expansion}
 
 \begin{multicols*}{2}
 \begin{expansion}[before=\vspace*{-11mm}]{navalbattles}
-  After Combat, return all Creature Bank Units back to the Creature Bank Units Cards pile.
-  If you win, you claim the Creature Bank's reward, as well as an additional reward for the Stacked Units you defeated.
-  Some rewards allow you to gain a Stacked Creature Bank Unit Card.
-  When you gain it, draw a random Stack Token and place it on that Unit.
+    After Combat, return all Creature Bank Units back to the Creature Bank Units Cards pile.
+    If you win, you claim the Creature Bank's reward, as well as an additional reward for the Stacked Units you defeated.
+
+    \subsection*{\pagetarget{Stack Tokens}{Stack Tokens}}
+        \setlength\intextsep{0pt}
+        \setlength\columnsep{1em}
+        \begin{wrapfigure}{r}{0.2\linewidth}
+            \includegraphics[width=\linewidth]{\images/stack_token.png}
+        \end{wrapfigure}
+    Each Stack Token modifies a Unit's statistics in the following way: +1~\svg{attack}, +1~\svg{defense}, +1~\svg{health_points}, or +2~\svg{initiative}.
+    Furthermore, each Unit with a Stack Token is now a \textbf{Stacked Unit}, which is similar to reinforced Faction Units: when it takes damage equal to or greater than its HP, instead of removing the Unit Card from the Combat Board, discard the Stack Token from that Unit and deal any leftover damage (if any), deducting it from the new HP.
```

Comment:
```suggestion
    Furthermore, each Unit with a Stack Token is now a \textbf{Stacked Unit}, which is similar to reinforced Faction Units: when it takes damage equal to or greater than its \svg{health_points}, instead of removing the Unit Card from the Combat Board, discard the Stack Token from that Unit and deal any leftover damage, deducting it from the new \svg{health_points}.
```
I think this text was already merged before, but "if any" is redundant here.